android/src/main/java/com/fidelreactlibrary/adapters/FidelOptionsAdapter.java

```diff
@@ -1,6 +1,7 @@
 package com.fidelreactlibrary.adapters;
 
 import android.graphics.Bitmap;
+import android.util.Log;
```

Comment:
This import is not needed anymore :) can be deleted.

Reply:
Sorry, forgot about that. I'll remove it.